docs/user/datastores/runtime_config.rst

```diff
@@ -0,0 +1,103 @@
+.. _geomesa_site_xml:
+
+Runtime Configuration
+=====================
+
+GeoMesa uses system properties for various runtime configuration options. As a convenience, properties
+can be specified in an XML file instead of the command line. When run, GeoMesa will load
+the file ``geomesa-site.xml`` from the classpath (if available), and use any properties configured there.
+
+To configure a GeoMesa tools distribution, place ``geomesa-site.xml`` in the ``conf`` folder.
```

Comment:
Wouldn't this now just be a "GeoMesa distribution"? Or maybe "GeoMesa binary distribution"?